lib/libxdp/tests/test_link_detach.c

```diff
@@ -11,7 +11,8 @@
 #include <string.h>
 #include <sys/resource.h>
 #include <unistd.h>
-
+#include <linux/bpf.h>
+#include <linux/if_link.h>
```

Comment:
Don't think these are strictly speaking needed? :)

Reply:
we don't need bpf.h but we need if_link.h for `XDP_FLAGS_SKB_MODE`